<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigconverterClass</class>
 <widget class="QMainWindow" name="ConfigconverterClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>867</width>
    <height>801</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configconverter</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPlainTextEdit" name="orgConfigText">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>411</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="dstConfigText">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>50</y>
      <width>411</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>  原配置</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>转换配置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="checkBtn">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>710</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>检查配置文件格式</string>
    </property>
   </widget>
   <widget class="QPushButton" name="convertBtn">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>370</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>开始转换</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>410</y>
      <width>461</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>批量转换</string>
    </property>
    <widget class="QPushButton" name="openfileBtn">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>40</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>打开文件夹</string>
     </property>
    </widget>
    <widget class="QPushButton" name="saveBtn">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>100</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>保存路径</string>
     </property>
    </widget>
    <widget class="QPushButton" name="startBtn">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>220</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>开始转换（特效）</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="openDirlineEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="saveDirLineEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="startBorderBtn">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>220</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>开始转换（边框）</string>
     </property>
    </widget>
    <widget class="QComboBox" name="typeCombox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>150</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>aaa</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>新建项目</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>新建项目</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>170</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>选择类型</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>867</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Configconverter.qrc"/>
 </resources>
 <connections/>
</ui>
